src/actions/effect/sepia.ts

```diff
@@ -1,19 +1,15 @@
 import Action from "../Action";
 import {IAction} from "../../interfaces/IAction";
+import {prepareEffect} from "./prepareEffect";
 
 /**
  * @implements IAction
  */
```

Comment:
Can we combine all Effect classes to a single class?

it seems like prepareEffect takes care of most of the heavy lifting.